tests/snmp_test.py

```diff
@@ -18,10 +22,15 @@ def ipv6_snmp_client(snmpsim, snmp_test_port) -> SNMP:
     return SNMP(device)
 
 
-@pytest.fixture(scope="session")
+@pytest.fixture()
```

Comment:
Why has the scope been removed?

Reply:
Per session ended up mocking the pysnmp commands for all tests, not just the ones that using the fixture